DESCRIPTION

```diff
@@ -22,7 +22,8 @@ BugReports:
     https://github.com/open-systems-pharmacology/OSPSuite.RUtils/issues
 Depends: 
     R (>= 3.6)
-Imports: 
+Imports:
+    purrr,
```

Comment:
We already use _purrr_ in the _ospsuite_ anyway, so this is **not** an additional dependency and shouldn't be an issue.